openedx/core/djangoapps/course_groups/tests/test_cohorts.py

```diff
@@ -172,6 +172,52 @@ def test_get_cohort_id(self):
             lambda: cohorts.get_cohort_id(user, SlashSeparatedCourseKey("course", "does_not", "exist"))
         )
 
+    def test_assignment_type(self):
+        """
+        Make sure that cohorts.set_assignment_type() and cohorts.get_assignment_type() works correctly.
+        """
+        course = modulestore().get_course(self.toy_course_key)
+        self.assertFalse(course.is_cohorted)
+
+        config_course_cohorts(course, discussions=[], cohorted=True)
```

Comment:
if the modulestore-based settings are going away, it seems wrong to use them to set up tests (except where we are testing the migration part).